sdk/include/platform/sunburst/platform-simulation_exit.hh

```diff
@@ -0,0 +1,25 @@
+// Copyright Microsoft and CHERIoT Contributors.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#ifdef SIMULATION
+#	include <stdint.h>
+#	include <platform-uart.hh>
+#	include <string_view>
+
+static void platform_simulation_exit(uint32_t code)
+{
+#	if DEVICE_EXISTS(uart0)
+	auto uart = MMIO_CAPABILITY(Uart, uart0);
+#	elif DEVICE_EXISTS(uart)
+	auto uart = MMIO_CAPABILITY(Uart, uart);
+#	endif
```

Comment:
Our normal style for this is to move the `auto UART=` out of the ifdef block. This makes it clearer that the existence of the variable is not conditional.